C-Plus-Plus/other/Kth largest element.cpp

```diff
@@ -0,0 +1,41 @@
+//Q Kth largest element 
```

Comment:
Have a beginner description about the Algorithms and the Code Implementation in a Multi-Line Comment.